observations/obs_converters/CONAGUA/README.rst

```diff
@@ -1,41 +1,64 @@
 CONAGUA
 =======
 
-The streamflow observations from CONAGUA are naturally in a Microsoft 
-database format. Mirce converts these one-at-a-time to a csv format.
-The filenames have a gage identifier in them, there is another file
-that has the lat/lon of the gage.
+Overview: 
+--------- 
+CONAGUA stands for Comision Nacional del Agua (National Water Commision).
+It is Mexico's central authority for water management.
+Established in 1989, CONAGUA is responsible for management, preservation,
+and distribution of water resources in Mexico.
+
+Data Source:
+------------
+The streamflow observations from CONAGUA are naturally in a Microsoft
+database format. These can be obtained from the
+`CONAGUA Webpage <https://www.gob.mx/conagua>`_.
+Mirce, a PhD student at the time from UNAM,  converts these
+one-at-a-time to a "csv" format. The filenames have a gauge
+identifier in them. There is also another file
+that has the lat/lon of the gauge.
 
 .. code-block:: text
 
    /glade/scratch/mirce/LaSierra/Observations/
```

Comment:
This directory doesn't exist:

ls /glade/scratch/mirce/LaSierra/Observations/
ls: cannot access '/glade/scratch/mirce/LaSierra/Observations/': No such file or directory

Reply:
Yes, it looks like the data has been scrubbed. I thought I'd keep it for legacy but maybe I shouldn't. I'll go ahead and remove that reference